draft-ietf-quic-manageability.md

```diff
@@ -945,68 +945,72 @@ deliver packets between server and NAT. Reducing the timeout on UDP NATs might
+{{?I-D.ietf-quic-applicability}}. Therefore it is often preferable for QUIC
+to fail and fallback during connectiion establishment rather persisting with a
+connection that might be unstable or expierencing
+sudden connection errors and time outs later during the connection.
 
 ### "Helping" with routing infrastructure issues
 
 Concealing client address changes in order to simplify operational routing
 issues will mask important signals that drive security mechanisms, and
 therefore opens QUIC up to various attacks.
 
-One challenge in QUIC deployments that want to benefit from QUIC's migration
-capability is server infrastructures with routers and switches that direct
-traffic based on address-port 4-tuple rather than connection ID. The use of
-source IP address means that a NAT rebinding or address migration will deliver
-packets to the wrong server. As all QUIC payloads are encrypted, routers and
-switches will not have access to negotiated but not-yet-in-use CIDs. This is a
-particular problem for low-state load balancers. {{QUIC_LB}} addresses this
-problem proposing a QUIC extension to allow some server-load balancer
-coordination for routable CIDs.
-
-It seems that a NAT anywhere in the front of such an infrastructure setup could
-save the effort of converting all these devices by decoding routable connection
-IDs and rewriting the packet IP addresses to allow consistent routing by legacy
-devices.
-
-Unfortunately, the change of IP address or port is an important signal to QUIC
-endpoints. It requires a review of path-dependent variables like congestion
-control parameters. It can also signify various attacks that mislead one
-endpoint about the best peer address for the connection (see section 9 of
-{{QUIC-TRANSPORT}}). The QUIC PATH_CHALLENGE and PATH_RESPONSE frames are
-intended to detect and mitigate these attacks and verify connectivity to the
-new address. This mechanism cannot work if the NAT is bleaching peer address
-changes.
+While QUIC's migration capability makes it possible for an server to survive
+address changes, this does not work if the routers or swtches in the server
+infrastructures reply on address-port 4-tuple as a NAT rebinding or address
+migration will cause packets to be delivered to the wrong server. {{QUIC_LB}}
+addresses this problem proposing a QUIC extension to allow server-load balancer
+coordination to routable CIDs.
+
+An alternative, potentially simpler approach seem to be the use of NAT in front
+of such an infrastructure setup, however, hinding information about the change
+of the IP address or port conceals important, and security relevant information
+from QUIC enpoint and as such would facilitate amplification attacks (see
+section 9 of {{QUIC-TRANSPORT}}).
+
+An NAT that blleaches peer address changes, hinders the other end
+to detect and mitigate these attacks and verify connectivity to the
+new address based on QUIC PATH_CHALLENGE and PATH_RESPONSE frames.
```

Comment:
This paragraph is probably redundant...

Reply:
Honestly, I prefer the original wording of this subsection. But you're the editor!

Reply:
I was trying to make it more clear what the recommendation actually is and remove some redundancy. I can give it another, less invasive try.

However, I think the main point about redundancy is actually that you basically say the same thing in this and the previous section (only that the motivation do use CID is different). So I would actually rather structure the text by "approach" than by use case but that would an even bigger rewrite. 

I'll have another look with a fresh pair of eyes tomorrow!